packages/pn-personafisica-webapp/src/components/Contacts/CancelVerificationModal.tsx

```diff
@@ -22,22 +23,24 @@ const CancelVerificationModal = ({ open, handleClose }: Props) => {
   };
 
   return (
-    <Dialog open={open} onClose={handleClose} data-testid="cancelVerificationModal">
-      <DialogTitle id="dialog-title" sx={{ pt: 4, px: 4 }}>
+    <PnDialog open={open} onClose={handleClose} data-testid="cancelVerificationModal">
```

Comment:
Checks the comments left on CodeModal about distance between Title and Content, and Content and Actions